recipes/emacs-26-pretest.rb

```diff
@@ -0,0 +1,25 @@
+recipe 'emacs-26-pretest' do
+  git 'http://git.savannah.gnu.org/r/emacs.git --branch emacs-26.0.90'
```

Comment:
Argh! There is a bug here for the from-source version: there should be a second parameter instead of a longer string containing the flag. Thought I had fixed that already, will do now.